selftests/functional/utils/distro.py

```diff
@@ -53,3 +53,14 @@ def test_rhel_9_1(self):
             + os.uname().machine.encode()
             + b") version 9 release 1\n",
         )
+
+    def test_debian_12_7(self):
+        """
+        :avocado: dependency={"type": "podman-image", "uri": "docker.io/library/python:3.12"}
```

Comment:
I am not sure if this is stable enough, right now it points to python3.12.6, but when the python3.12.7 will be release this container will change and the Debian version may change as well, am I right?

Reply:
Good point, I'll look into pinning that into something more stable.

Reply:
I've replaced that URI with one with a digest reference, which is unique and immutable and points to a container using a Debian 12.7 distro.